crypter.go

```diff
@@ -74,8 +75,32 @@ type recipientKeyInfo struct {
 // EncrypterOptions represents options that can be set on new encrypters.
 type EncrypterOptions struct {
 	Compression CompressionAlgorithm
-	Type        ContentType
-	ContentType ContentType
+
+	// Optional map of additional keys to be inserted into the protected header
+	// of a JWS object. Some specifications which make use of JWS like to insert
+	// additional values here. All values must be JSON-serializable.
+	ExtraHeaders map[HeaderKey]interface{}
```

Comment:
You mention ContentType/Type wasn't tagged yet, but I think ContentType was, but not Type. I can see ContentType in EncrypterOptions here: https://godoc.org/gopkg.in/square/go-jose.v2. It's fine either way though.